tools/azsdk-cli/Azure.Sdk.Tools.Cli/Services/OpenAIService.cs

```diff
@@ -0,0 +1,61 @@
+using System.Text.Json;
+using Azure.AI.OpenAI;
+using Azure.Core;
+using Azure.Identity;
+using Azure.Sdk.Tools.Cli.Models;
+using OpenAI.Chat;
+
+namespace Azure.Sdk.Tools.Cli.Services;
+
+/// <summary>
+/// Interface for Azure OpenAI client to enable dependency injection and testing
+/// </summary>
+public interface IAzureOpenAIClient
```

Comment:
This file should should be deleted.   You shouldn't be adding an interface facade over Azure clients, not building/configuring them separately.   

Please use the Azure client builder, as mentioned in `Program.cs`

Reply:
Deleted!